src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+The further consequence of this assumption is that logic and its subject matter (here the world) are utterly different from each other; "each turns out to be a sphere divorced from the other" (Hegel, 2010, 24/21.29). As Hegel notes, this leads exactly to the Kantian position of thing-in-itself, since logic or thought are utterly other to the world, the world is a complete thing unto itself logically prior it being given as material for cognition as intuitions. In other words, the world cannot be thought of with any certainty prior to its experience; or, put differently still, thought is incapable of coming to any resolution about its subject matter (be that the world, existence or anything).
+
+If one pursues this line of thought further, then it effectively means that _thought is incapable of coming to any conclusions about itself_. Thought, as well as logic, are merely the formal instruments of experience that await reception of worldly material; prior to which thought is unable to think. Thought and logic in this view are _externally determined_, not _self-determining_. Put bluntly, then, according to this perspective thought cannot think. One does not make up one's own mind about something, but only ever reacts to certain sensuous data.
+
+A further conclusion of this thread is that the world itself, or any subject matter of logic, is inherently _illogical_. This is the other side of that stark difference between logic and its subject matter. Since logic is mere form and this form requires material, the world cannot with any definitive resolution be understood to be logical _prior_ to its reception as material for cognition. We cannot speak of a reality outside experience. Hence, one is firmly placed in the Kantian position that posits the world _as such_ to be a _thing-in-itself_.[^1]
+
+[^1]: It can be seen that just as thought is downstream from pure concepts and sensuous intuitions, the whole logic of the argument is itself heteronomous. Just as thought must assume worldly material given to it, so the transcendental argument itself turns around the world as given, despite positing its source as irrational. (Niklas)
+
+Hegel notes that the manner one here views logic and its subject matter, thought and the world, subject and object, is in _phenomenal_ terms. But he calls it a prejudice to carry over this view into reason and logic. The argument is incoherent from the start since the phenomenal is allowed to make a metaphysical claim, but the former is exactly not the latter. The way one experiences things through the senses cannot be the standard for how everything is evaluated. Indeed, senses do not in the strict sense evaluate. This is not to say that experience is not real or important, but it is being misused if it is taken out of its scope. This miscarriage of logic is simply classed as an error and Hegel notes how it can be an impediment to philosophical thinking. 
```

Comment:
I don't think it's clear why the phenomenal is allowed to make a metaphysical claim

Reply:
It's not, that's the point. But I see that this isn't well expressed so I've added a bit to help.